### DISCUSSION TITLE
Project Template support for Advanced Setup

### DISCUSSION THREAD

**Discussion by zachhutchins** [mia-platform/community#525] [Q&A] [answered]
### What product are you having troubles with?

Console

### What Console version are you using?

1.102.8

### Description

From the documentation there does not appear to be any way to add advanced configuration to project templates. 

We have need for adding default http filters for api-gateway.

Is this something that is currently supported in project templates? Should I use a seperate file other than `api-console-config.json` to define the files?

### Actual Outcome

I was not able to add advanced configuration in my project template.

### Expected Outcome

I can add advanced configuration in my project template.

**Comment by fredmaggiowski** [comment, marked as answer]
Hi @zachhutchins, at the moment there is no support for advanced configurations in project templates, especially when using the Enhanced Workflow, if this is an interesting feature for you reach out your Mia-Platform referent to open a feature proposal
